source/docs/common-mechanisms/linear-motion-guide/rigging.rst

```diff
@@ -57,25 +65,39 @@ When the motor rotates one direction, the extension spool reels in the first str
 
 A cascaded system can be retracted in three ways: using continuous retraction, elastic retraction, or reverse-cascade retraction.
 
+.. figure:: images/rigging/cascade-extension.png
+   :alt: A diagram of cascade rigging
+
+   Cascade rigging diagram
+
 Retraction Options
-^^^^^^^^^^^^^^^^^^
+------------------
 
 Continuous retraction
-~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^
+
+As the name suggests, continuous retraction utilizes inverted continuous rigging to retract the slides. There are two main methods of doing this, **free floating retraction** where a string is run from the last stage directly back to the spool, and **with slide retraction** where the string is run in an inverse continuous pattern. Generally, **with slide retraction** should be used with extensions going outside the robot frame to prevent entanglement, and **free floating retraction** should only be used with purely vertical slides.
+
+.. note:: If continuous retraction is being used with cascade extension, the two spools cannot be the same diameter. If the variable :math:`N` is the number of stages in the system, the diameter of the cascade extension spool must be :math:`N` **times smaller** than the continuous retraction spool.
+
+.. figure:: images/rigging/continuous-retraction.png
+   :alt: A diagram of continuous retraction
+
+   Continuous Retraction
 
-As the name suggests, continuous retraction utilizes the same retraction method as continuous rigging with one notable difference: if the variable :math:`N` is the number of stages in the system, the diameter of the extension spool must be :math:`N` **times smaller** than the retraction spool.
+.. warning:: If free floating retraction is used, **make sure your retract string is always parallel to the slides**. If the string is misaligned, it can pull on your slide and cause excess bending forces on the stages.
 
 Advantages:
 
-- Requires less string
-  (less strings to tension, less strings that can become loose)
+- Simplest retraction rigging
 
 Disadvantages:
 
-- Requires more space, as it demands a larger spool retraction spool
+- Can become tangled if free floating
+- Requires different sized spools if used with cascade retraction
 
 Elastic retraction
-~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^
 
 Instead of retracting using a retraction spool, one common way to retract is to attach a piece of elastic (commonly surgical tubing) to the last stage. The elastic applies a force on the last stage that is counteracted by the motor when extending. However, when retracting, the motor reels the last slide back in. While this is the retraction method recommended by many kit slide manuals, we do not recommend this method.
```

Comment:
```suggestion
Instead of retracting using a retraction spool, one common way to retract is to attach a piece of elastic (commonly surgical tubing) to the last stage. The elastic applies a force on the last stage that is counteracted by the motor when extending. However, when retracting, the motor reels the last slide back in. While this is the retraction method recommended by many kit slide manuals, this method is not recommended.
```